client/components/item/AddItem.js

```diff
@@ -51,53 +51,35 @@ export const AddItem = ({
    *
    * @return {type} description of return value
    */
-  const handleSubmit = () => {
-    console.log('initial', initialData);
+  const handleSubmit = (data) => {
+    console.log('initial', data);
     if (isEdit) {
       if (packId && initialData.global) {
+        try {
+          dispatch(editItemsGlobalAsDuplicate(data));
+          closeModalHandler();
+        } catch (error) {
+          console.log(error);
+        }
         console.log('editing', packId);
-
-        dispatch(
-          editItemsGlobalAsDuplicate({
-            itemId: _id,
-            packId,
-            name,
-            weight,
-            quantity,
-            unit,
-            type: categoryType,
-          }),
-        );
-        closeModalHandler();
       } else {
-        dispatch(
-          editPackItem({
-            name,
-            weight,
-            quantity,
-            unit,
-            type: categoryType,
-            _id: initialData._id,
-          }),
-        );
-        setPage(1);
-        closeModalHandler();
-        setRefetch(refetch !== true);
+        try {
+          dispatch(editPackItem(data));
+          setPage(1);
+          closeModalHandler();
+          setRefetch(refetch !== true);
+        } catch (error) {
+          console.log(error);
```

Comment:
shouldn't we set the errors like you did in other places you added try catch ?

- [ ] set errors not just log. could probably move to helper to minimize redundancy

Reply:
the other inputs arent setting errors